Branch-SDK-Automation-TestBed/src/main/java/io/branch/saas/sdk/testbed/activities/GenerateUrlActivity.java

```diff
@@ -145,6 +145,24 @@ public boolean onChannelSelected(String channelName, BranchUniversalObject buo,
                             return false;
                         }
                     });
+        } else if (clickType.equals(Constants.CREATE_NATIVE_SHARE_LINK)) {
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP_MR1) {
+                Branch.getInstance().share(this, Common.branchUniversalObject, Common.lp, new Branch.BranchNativeLinkShareListener() {
+                            @Override
+                            public void onLinkShareResponse(String sharedLink, BranchError error) {
+                                Log.d("Native Share Sheet:", "Link Shared: " + sharedLink);
+                            }
+
+                            @Override
+                            public void onChannelSelected(String channelName) {
+                                Log.d("Native Share Sheet:", "Channel Selected: " + channelName);
+                            }
+
+                        },
+                        "Sharing Branch Short URL", "Using Native Chooser Dialog");
```

Comment:
Should title be left as an api? I don't think we should leave `Branch` in the messaging.

Reply:
Should I update it to title -  "Native Share Sheet:" and subject -  "Link Shared: " ?